Formula/tbb.rb

```diff
@@ -19,50 +17,74 @@ class Tbb < Formula
   depends_on "swig" => :build
   depends_on "python@3.9"
 
-  # Remove when upstream fix is released
-  # https://github.com/oneapi-src/oneTBB/pull/258
-  patch do
-    url "https://github.com/oneapi-src/oneTBB/commit/86f6dcdc17a8f5ef2382faaef860cfa5243984fe.patch?full_index=1"
-    sha256 "d62cb666de4010998c339cde6f41c7623a07e9fc69e498f2e149821c0c2c6dd0"
-  end
+  # Fix installation of Python components
+  # See https://github.com/oneapi-src/oneTBB/issues/343
+  patch :DATA
 
   def install
-    compiler = (ENV.compiler == :clang) ? "clang" : "gcc"
-    system "make", "tbb_build_prefix=BUILDPREFIX", "compiler=#{compiler}"
-    lib.install Dir["build/BUILDPREFIX_release/*.dylib"]
+    args = *std_cmake_args + %w[
+      -DTBB_TEST=OFF
+      -DTBB4PY_BUILD=ON
```

Comment:
Add
```
-DCMAKE_INSTALL_RPATH=#{lib}
```
?

Reply:
Actually, this doesn't look like the problem. I think we need to pass an `rpath` to the linker in the Python build. There's another formula that does this; if I can only remember which one 😅

Reply:
Try this

https://github.com/Homebrew/homebrew-core/blob/8bfc48e6caa277cd8680bf5e8931e31de22d62ef/Formula/coin3d.rb#L67-L70

Reply:
Thanks, let's give it a try

Reply:
It worked 🎉  Still some other failures though